docker/secondary-analysis-python/tools/secondary_analysis/secondary_analysis/utils.py

```diff
@@ -0,0 +1,43 @@
+import requests
+import logging
+import time
+
+
+def get_file_by_uuid(file_id, dss_url):
```

Comment:
If possible it would be great to throw some docstrings on these with a short description + parameter types. 

It _really_ makes it easier to go back in and see what the variable types are. :)